README.md

```diff
@@ -17,12 +17,24 @@ Python packages with our recommended tooling set up and ready to go.
 > package template (e.g., [`scikit-hep`](https://github.com/scikit-hep/cookie)),
 > we recommend using their template instead of this one.
 
+## Scope and contributing
+
+This project was created by [research software engineers] at [UCL's Centre for Advanced Research Computing][UCL ARC].
+We made it with our own research software projects in mind - we don't explicitly support external users, but whoever you are, we hope you might find it useful!
+
+As such, contributions to this repository are only open to members ARC - if you're a member of ARC, see [CONTRIBUTING.md] for more info on how to contribute.
```

Comment:
I don't like this at all

Reply:
Why?

Reply:
If someone outside ARC has the impetus to write a PR to try and improve our tool, I will be very willing to review that.

Reply:
It was more a question for @paddyroddy, since simply writing

> I don't like this at all

is not a helpful review 😉

Reply:
Also this feels unnecessarily unwelcomming. If we have floods of contributions, maybe let's think again. But the odd PR from --say-- a UCL student or external researcher or a _chof cough_ RSE from Manchester, is very welcome.

Reply:
@dstansby I thought it was obvious in context, but yes, basically what @samcunliffe said.